js/ui/windowManager.js

```diff
@@ -2224,6 +2224,11 @@ var WindowManager = new Lang.Class({
                 let parent = window.get_transient_for();
                 if (parent)
                     this._checkDimming(parent);
+
+                if (actor._animatableSurface) {
+                    this._animationsServer.unregister_surface(actor._animatableSurface);
+                    actor._animatableSurface = null;
+                }
```

Comment:
Can you confirm this isn't necessary at https://github.com/endlessm/gnome-shell/pull/329/commits/a11c8dd72d3fae0748c2aa990ed1645d54c1df9c#diff-3411cdb80aaaddfcaa2c6d036a436b69L2461 as well?

Reply:
It is not necessary to also have it there, because the `unmanaged` signal I connect to is emitted for every window where the effect is registered in the first place.